pages/members/[id]/index.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
 import { useRouter } from 'next/router';
 import Profile from 'components/member-profile';
-
+import Layout from 'components/layout';
 const MemberProfile = () => {
   let { query } = useRouter();
-  return <Profile id={query.id} />;
+  const { first_name, last_name } = query;
+
+  return (
+    <Layout
+      title={`${
+        query.first_name && query.last_name ? `${first_name} ${last_name}` : query.id
```

Comment:
Could we have this a bit cleaner please?

Something like
```js
const { first_name: firstName, last_name: lastName, id: queryID } = query;
const hasBothNames = firstName && lastName;

const titleStr = hasBothNames ? `${firstName} ${lastName}` : queryID
```

Reply:
Spreading the query object is ok, But Sahil(@invalidtoken ) has used `first_name` and `last_name` in snake_case, so I just kept it that way.
Should I convert all the `first_name` and `last_name`s to camelCase?, If yes then we also won't need to rename object keys.

Reply:
Backend replies in snake_case, in JS, we adapt it to camelCase (this helps in quickly assessing if the data is coming from an API, or our JS world)

Will add this in the code docs